docs/products/clickhouse/howto/integrate-kafka.rst

```diff
@@ -0,0 +1,164 @@
+
+.. note::
+
+    ClickHouse service integrations are available for Startup plans and higher.
+
+Prerequisites
+-------------
+
+You will need:
+
+* Aiven for ClickHouse service
+* Aiven for Apache Kafka service or a self-hosted Apache Kafka service
+* At least one topic in the Apache Kafka service
+
+Variables
+-------------
+
+The following variables will be used later in the code snippets:
+
+============================     ==========================================================================================================
+Variable                         Description
+============================     ==========================================================================================================
+``CLICKHOUSE_SERVICE_NAME``      Name of your Aiven for ClickHouse service.
+``KAFKA_SERVICE_NAME``           Name of the Apache Kafka service you use for the integration.
+``PROJECT``                      Name of Aiven project where your services are located.
+``CONNECTOR_TABLE_NAME``         Name of the Kafka engine virtual table that is used as a connector.
+``DATA_FORMAT``                  Input/output data format in which data is accepted into ClickHouse. See :doc:`../reference/supported-input-output-formats`
+``CONSUMER_GROUP_NAME``          Name of the consumer group. Each message is delivered once per consumer group.
+============================     ==========================================================================================================
+
+Create an integration
+----------------------
+
+You can create an integration with the help of :ref:`Aiven CLI <avn_service_integration_create>`, or through the `Aiven console <https://console.aiven.io/>`_ by following these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse service.
+#. In the *Overview* tab, scroll till the section *Service integrations* and click on **Set up integration**. A modal window with available integration options will appear.
+#. Select Apache Kafka from the list of the options.
+#. Choose your service from the list. This is also a place where you can trigger creation of a new Aiven for Apache Kafka service.
+#. Click on **Enable**.
+#. Your Apache Kafka service will be added to the list of enabled service integrations.
+#. You can now close *Service integrations* modal window.
+#. The Kafka integration table will now appear in the database.
+
+.. note::
+
+    Creation of this default configuration only creates an empty ClickHouse but does not create any table. This database is named after the Kafka service.
+
+
+Integration settings
+-----------------------
+
+Next step is to configure the topic and data format options for the integration. This will create a virtual table inside ClickHouse that can receive and send messages from multiple topics. You can have as many of such tables as you need. You need to define for each table
```

Comment:
It can both send and receive (not sure if it is a good practice though, sending data back to Kafka seems to be a rare use case).

However, writing to more than one topic is not supported. I've added a warning for it